protocol.html

```diff
@@ -469,6 +473,8 @@ <h4 property="schema:name skos:prefLabel">Classes of Products</h4>
                       <span rel="skos:hasTopConcept"><span resource="#SubscriptionClient"></span><span resource="#SubscriptionServer"></span><span resource="#NotificationSender"></span><span resource="#NotificationReceiver"></span><span resource="#ResourceServer"></span></span>
 
                       <dl>
+                        <dt about="#ResourceServer" property="skos:prefLabel" rel="skos:broadMatch" resource="spec:RespondingAgent" typeof="skos:Concept"><dfn id="ResourceServer">Resource Server</dfn></dt>
+                        <dd about="#ResourceServer" property="skos:definition">A <em>resource server</em> that builds on HTTP <cite>server</cite> [<cite><a class="bibref" href="#bib-rfc7230">RFC7230</a></cite>] and [<cite><a class="bibref" href="#bib-rfc7231">RFC7231</a></cite>] by defining media types, HTTP header fields, and the behaviour of resources, as identified by link relations.</dd>
```

Comment:
```suggestion
                        <dd about="#ResourceServer" property="skos:definition">A <em>resource server</em> builds on an HTTP <cite>server</cite> [<cite><a class="bibref" href="#bib-rfc7230">RFC7230</a></cite>] and [<cite><a class="bibref" href="#bib-rfc7231">RFC7231</a></cite>] by defining media types, HTTP header fields, and the behaviors of resources, as identified by link relations.</dd>
```

Reply:
+1 to removing "that"
-1 to US English spelling; should be "behaviours"

Reply:
Adjusted.